### DISCUSSION TITLE
Half-width bar

### DISCUSSION THREAD

**Discussion by hamadahiro** [FelixKratz/SketchyBar#434] [Q&A] [answered]
Is it possible to make the SketchyBar just half the width of the screen, so that it covers the left side of the menu bar and exposes the icons on the right side of macOS, allowing them to be clicked on?

**Comment by FelixKratz** [comment, marked as answer]
Check #221 in combination with the `topmost` bar property you should be able to achieve this.

**Reply by hamadahiro** [reply]
Thank you. I've tried this, but I can't seem to make only one bracket transparent. When I set the color of the bar to transparent and then add color only to the left side, leaving the right side transparent, it does nothing. I can only change the color for the whole bar.

```
sketchybar --add braket left_items ${LEFT_ITEMS[@]} \
           --set left_items color=0xff000000 \
                            background.height=$BAR_HEIGHT

sketchybar --add braket right_items ${RIGHT_ITEMS[@]} \
           --set right_items color=0x00000000 \
                             background.height=$BAR_HEIGHT
```

Can you confirm whether this is possible at all? If yes, would you mind explaining how to achieve this? Thank you.

**Reply by FelixKratz** [reply]
This is possible, there is a slight problem in your code, since you need to use `background.color` instead of `color` for the brackets.

You could also completely leave out the right bracket if you only want items to appear on the left side or vice versa.

Try running sketchybar from the commandline to see the errors like this printed to screen:
```bash
brew services stop sketchybar
sketchybar
```
